package.json

```diff
@@ -2,7 +2,7 @@
   "name": "kentcdodds.com",
   "version": "1.0.0",
   "engines": {
-    "node": "18",
+    "node": "16",
```

Comment:
I just noticed this change. Is this intentional?

Reply:
Oh no that wasn't intentional, sorry

You can revert that if you want

Reply:
No worries :) Thanks!